### PR TITLE
Create python-app.yml

### PR DESCRIPTION
**Pull request by P0NDER0SA**

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```
Comment (var-kyle) [inline .github/workflows/python-app.yml]: I think we should use the same Python version as the project to avoid any possible incompatibilities.

**Review by var-kyle — CHANGES_REQUESTED** [review]

File: .github/workflows/python-app.yml (hunk 1 above)
Comment (DaceyTom2) [reply .github/workflows/python-app.yml]: I agree python-version: 3.7.2 would be ideal

Comment (P0NDER0SA) [reply .github/workflows/python-app.yml]: sounds good boys!  nice catch.

**Comment by P0NDER0SA** [comment]
The best we can do is 3.7.x which works.  i think it might actually be faster too.

**Review by DaceyTom2 — APPROVED** [review]
Looks good

**Review by var-kyle — APPROVED** [review]
Looks good!

**Pull request merged**